### PR TITLE
add php cassandra driver layer

### PR DESCRIPTION
**Pull request by valentijnscholten**
This PR adds the Datastax PHP Cassandra Driver. It's built from source as the driver provided by Datastax is outdated and not supported / working with PHP 7.2+.

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you. Could you also update the readme?

**Comment by valentijnscholten** [comment]
Done, I knew I was forgetting something.

File: Readme.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ extension=/opt/bref-extra/amqp.so
 | ---- | ----------------------------| -------------- |
 | AMQP | `${bref:extra.amqp-php-74}` | `extension=/opt/bref-extra/amqp.so` |
 | Blackfire | `${bref:extra.blackfire-php-74}` | `extension=/opt/bref-extra/blackfire.so` |
+| Caansdra | `${bref:extra.cassandra-php-74}` | `extension=/opt/bref-extra/cassandra.so` |
```
Comment (Nyholm) [inline Readme.md:48]:
```suggestion
| Cassandra | `${bref:extra.cassandra-php-74}` | `extension=/opt/bref-extra/cassandra.so` |
```

**Comment by Nyholm** [comment]
thank you

**Pull request merged**